File: Contests/Codeforces/CodeforcesRound#841/B.py
t = int(input())

for k in range(0, t):
    n = int(input())
    ans = (n*(n+1)*(2*n + 1)/6) * 2022
    ans = ans + (((n-1) * n * (n+1))/3) * 2022
    # ans = ans % (10000000007)

    print(int(ans))

File: Contests/Codeforces/CodeforcesRound#841/B.cpp
#include <bits/stdc++.h>

using namespace std;
#define int long long

const int mod = 1e9+7;

int fExp(int a, int b){
    int ans = 1;

    while(b){
        if(b&1) ans = (ans * a) % mod;

        a = a*a %mod;
        b >>= 1;
    }

    return ans;
}

int32_t main(){
    int t;
    cin >> t;

    while(t--){
        int x;
        cin >> x;

        int ans = ((((((x * (x+1))%mod) * (2*x + 1)%mod)*fExp(6, mod - 2))%mod)*2022) % mod;
        ans = (ans + (2022*(((((x*(x-1)) % mod) * (x+1)%mod) * fExp(3, mod - 2))%mod) % mod)%mod) % mod;

        cout << ans << "\n";
    }


    return 0;
}